docs/spec/transactions/mark.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "docs/spec/transactions/mark.json",
+    "type": ["object", "null"],
+    "description": "A flat mapping of a user-defined mark.",
```

Comment:
@jahtalab maybe you can come up with a better description here.

Reply:
@simitt 
I think I should be something like: 
1. "A mark captures the timing of a significant event. They're organized into groups and can be set by the user or by the agent."
2. "A mark captures the timing of a significant event during the lifetime of a transaction. They can be set by the user or by the agent."

The second one doesn't specify the grouping since I think it is a technical detail.
Also, maybe we should use the same definition in both places.